platform/solana/client.go

```diff
@@ -0,0 +1,38 @@
+package solana
+
+import (
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+)
+
+const stakeProgramId = "Stake11111111111111111111111111111111111111"
+
+type Client struct {
+	blockatlas.Request
+}
+
+func (c *Client) GetCurrentVoteAccounts() (validators []VoteAccount, err error) {
```

Comment:
besides staking, can we support `TxAPI interface` too? 
```go
TxAPI interface {
	Platform
	GetTxsByAddress(address string) (TxPage, error)
}
```

Reply:
Our nodes do not index transactions, so we don't have a TxsByAddress rpc available. Would need a client to poll `getConfirmedBlock` and build a db to support this.

Reply:
I see, can we return empty list then?

Reply:
Sure, I'll add that